manubot/pandoc/cite_filter.py

```diff
@@ -195,7 +216,8 @@ def process_citations(doc: pf.Doc) -> None:
         req_cache.close()
 
     citations.write_citekeys_tsv(path=doc.get_metadata("manubot-output-citekeys"))
-    citations.write_csl_json(path=doc.get_metadata("manubot-output-bibliography"))
+    citations.write_csl_items(path=doc.get_metadata("manubot-output-bibliography"))
+    citations.write_csl_items(path=doc.manubot["bibliography_cache"])
```

Comment:
I may not be following the behavior here.  All of the citations are written to both the output bibliography and the bibliography cache, overwriting the existing bibliography cache?

Reply:
`manubot-output-bibliography` and `manubot-bibliography-cache` and both default to None. If both are specified, two bibliography files can be written. I don't think most users will specify both.

Will improve docs.